clusterlib/scheduler.py

```diff
@@ -24,6 +24,52 @@
 ]
 
 
+def _which(program):
```

Comment:
Why not using [shutil.which](https://docs.python.org/3.3/library/shutil.html#shutil.which) in python 3?